cmd/kubeadm/app/cmd/cmd.go

```diff
@@ -93,5 +111,60 @@ func NewKubeadmCommand(f cmdutil.Factory, in io.Reader, out, err io.Writer) *cob
 	experimentalCmd.AddCommand(phases.NewCmdPhase(out))
 	cmds.AddCommand(experimentalCmd)
 
+	cmds.SetHelpCommand(newKubeadmHelpCmd(cmds.Name()))
+	cmds.SetUsageTemplate(cmds.UsageTemplate() + kubeadmUsageTemplateRequiredImages)
 	return cmds
 }
+
+// newKubeadmHelpCmd returns a help command for kubeadm root command.
+// see cobra.Command#initHelpCmd() (command.go)
+func newKubeadmHelpCmd(cmdName string) *cobra.Command {
+	return &cobra.Command{
+		Use:   "help [command]",
+		Short: "Help about any command",
+		Long: `Help provides help for any command in the application.
```

Comment:
at the very begining, @jbeda suggested adding `kubeadm list-images`, here `list-images` actually is one of kubeadm's subcommands. about one or two days ago ,i talked this `list-images` with @luxas, he suggest `kubeadm help required-images`, and i guess he'd like we shouldn't add one extra kubeadm subcommand just for printing required images. so in my implementation, required-images is actually not subcommand. that's why ` kubeadm help cmd` diffs from `kubeadm cmd --help`, maybe we should call it `topic` instead of `cmd` ?